modules/ROOT/partials/toolbar-button-ids/advtable-toolbar-buttons.adoc

```diff
@@ -1,6 +1,10 @@
+[cols="d,a"]
```

Comment:
Likewise here, I'm not sure if we want this?

Reply:
Done